index.js

```diff
@@ -273,7 +273,7 @@ async.parallelPlus = function(functions, callback) {
     }
     var newFunctions = {};
     for (var func in functions) {
-        if (Object.prototype.hasOwnProperty.call(functions, func)) {
+        if (functions.hasOwnProperty(func)) {
```

Comment:
ESLint was getting mad at me for doing this. Thoughts? http://eslint.org/docs/rules/no-prototype-builtins